keyboards/tweetydabird/lotus58/nanoboot/rules.mk

```diff
@@ -0,0 +1,2 @@
+BOOTLOADER = qmk-hid
```

Comment:
Could be on json too (the bootloader part)

Reply:
Respectfully, but no.

Once the Bootloader size is able to be ported over to info.json, I will happily change this. Until then, I categorically refuse.
I understand the reasoning that info.json is the superior choice, and what we are striving too.

But in this case, with two that intertwined options in two different files in two different formats, it just causes confusion. You change them as a whole, or you do not change them. So this one stays for now.